ONSdigital/dp-legacy-cache-api
# Get Cache-Time Endpoint

### What

Added new `GetCacheTime` endpoint with 200 response.

### How to review

In order to check the endpoint is correctly setup, you will need to: 

1. Have mongodb shell installed (https://www.mongodb.com/docs/mongodb-shell/install/)
2. Run mongodb in a docker container (use command from readme).
3. Open a new terminal and enter `mongosh` - Your terminal should now be prefixed with 'test'
4. Then insert a document into our datasets collection by running: 
`db.datasets.insertOne({
    _id: “123”, 
    path: "hello",
    etag: "testetag",
    collectionId: 1234,
    releaseTime: new Date("2023-12-12T00:00:00Z")})`
5. Run the app with `make debug`
6. Using postman or similar, make a GET request to `localhost:29100/v1/cache-times/123`
7. You should see a 200 status code with the above inserted document in the response.

### Who can review

A member of the ONS
